packages/flutter/lib/src/painting/box_decoration.dart

```diff
@@ -54,6 +54,11 @@ import 'image_provider.dart';
 /// ```
 /// {@end-tool}
 ///
+/// Note that the [shape] or the [borderRadius] won't clip the children of the
```

Comment:
nit: our style guide asks to avoid "note that" formulations. Can we re-word this to just "The [shape] or the [borderRadius] won't clip..."?